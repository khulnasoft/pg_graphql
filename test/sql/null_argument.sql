begin;
    -- Test that the argument parser can handle null values

    create table account(
        id serial primary key,
        email text
    );

    select graphql.resolve($$
    mutation {
      insertIntoAccountCollection(objects: [
        { email: null }
      ]) {
        affectedCount
        records {
          id
          email
        }
      }
    }
    $$);

rollback;
