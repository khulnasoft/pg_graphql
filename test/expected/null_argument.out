begin;
    -- Test that the argument parser can handle null values
    create table account(
        id serial primary key,
        email text
    );
    select graphql.resolve($$
    mutation {
      insertIntoAccountCollection(objects: [
        { email: null }
      ]) {
        affectedCount
        records {
          id
          email
        }
      }
    }
    $$);
                                                resolve                                                 
--------------------------------------------------------------------------------------------------------
 {"data": {"insertIntoAccountCollection": {"records": [{"id": 1, "email": null}], "affectedCount": 1}}}
(1 row)

rollback;
